Example for creating a WiFi-PSK connection profile

bernhardkaindl opened the pull request:
Example to create a new WiFi-PSK network connection profile

igo95862 commented on examples/block/add-wifi-psk-connection.py:
```diff
@@ -0,0 +1,117 @@
+from sdbus_block.networkmanager import NetworkConnectionSettings
+from typing import Any, Dict, Tuple
+
+NetworkManagerConnectionProfile = Dict[str, Dict[str, Tuple[str, Any]]]
```
Hmm. I should probably add this as a type.

bernhardkaindl replied:
In new code, I composed the profile type in step because I found use of the "Group" (e.g.`ipv4`, `ipv6`, etc) and the individual settings tuple as well:
```py
NetworkSetting = Tuple[str, Any]
NetworkSettingsGroup = Dict[str, NetworkSetting]
NetworkSettingsProfile = Dict[str, NetworkSettingsGroup]
```
Here I used shorter names, but you can use the longer form and consumer of the lib can alway shorten these when importing them using `import` ... `as` ... like I did when I wanted a shorter
name for the type in my own code. You likely know best.

igo95862 replied:
I am going to try adding those types. Not sure how sphinx will react to it.

bernhardkaindl replied:
@igo95862: I pushed providing (and using) those type aliases as #7 now.
Sphinx unfolds the type alias to `Dict[str, Dict[str, Tuple[str, Any]]]`: There is no change in docs.

bernhardkaindl commented:
@igo95862 I converted it to use:
* The new type alias `NetworkManagerConnectionProperties`
* The new helper method `NetworkManagerSettings().get_connections_by_id(connection.id)`

I also added fixing a mypy warning and two sourcery warnings in another example I added before.
